### PR TITLE
More migration stress

### PR DESCRIPTION
**Pull request by chriscool**
Add revert tests to t0060-migration-stress.sh and also add an EXPENSIVE prereq to make them faster in the normal case.

### CODE REVIEW

**Comment by chriscool** [comment]
There is this failure:

```
expecting success: 

	comm -23 start_refs after_revert_refs > missing_refs &&
	test_cmp missing_refs empty_refs_file

> diff -u missing_refs empty_refs_file
--- missing_refs	2016-03-24 09:38:36.927208538 +0000
+++ empty_refs_file	2016-03-24 09:38:32.932312062 +0000
@@ -1 +0,0 @@
-QmemK5Vqg4Cao5otFSebPm5CBEHfuQmE2KLbxHGA11Gm3p

not ok 48 - refs look right
```

@whyrusleeping could you take a look at it. It seems to me that it might be a bug.

**Comment by whyrusleeping** [comment]
@chriscool this seems normal. The `start_refs` file contains refs before the initial migration. After the migration occurs, we run a GC. which means we drop refs that arent pinned.

What we should do here is, after the tests that i ran, but before the revert we should add a bunch more random content (and maybe a few pins too), collect the refs into a new `start_refs` file, and then run the test after the revert.

**Comment by chriscool** [comment]
@whyrusleeping ok I just pushed an amended commit with the changes you suggest and it works on my machine!

File: sharness/t0060-migration-stress.sh [hunk 1]
```diff
@@ -179,6 +193,91 @@ test_expect_success "no pinned objects are missing from local refs" '
...
+test_expect_success "'ipfs-2-to-3 -revert' succeeds" '
+	drun "$GUEST_IPFS_2_TO_3 -revert -path=/root/.ipfs" >actual
+'
+
+test_expect_success "'ipfs-2-to-3 -revert' output looks good" '
+	grep "writing keys:" actual ||
+	test_fsh cat actual
+'
+
+test_install_version "v0.3.10"
```
Comment (whyrusleeping) [inline sharness/t0060-migration-stress.sh]: should probably use `v0.3.11` here. since its the latest version

Comment (chriscool) [inline sharness/t0060-migration-stress.sh]: Ok I will change that.

**Comment by chriscool** [comment]
Now using v0.3.11 everywhere.

**Comment by whyrusleeping** [comment]
Cool, this looks great to me!

**Pull request merged**